cmd/migration-assist/commands/mysql.go

```diff
@@ -65,6 +68,24 @@ func runSourceCheckCmdF(cmd *cobra.Command, args []string) error {
 	}
 	baseLogger.Println("connected to mysql successfully...")
 
+	applied, err := mysqlDB.GetAppliedMigrations(cmd.Context())
+	if err != nil {
+		return fmt.Errorf("could not get applied migrations: %w", err)
+	}
+	mysqlConfig := store.DBConfig{
+		AppliedMigrations: applied,
+	}
+
+	b, err := json.MarshalIndent(mysqlConfig, "", "    ")
+	if err != nil {
+		return fmt.Errorf("could not marshal mysql config: %w", err)
+	}
+
+	outputFile, _ := cmd.Flags().GetString("output")
+	if err = os.WriteFile(outputFile, b, 0600); err != nil {
```

Comment:
1. Suggest making it `0666` to keep things simple.
2. Also add additional check to see if the file exists, and then prompt the user before going ahead and overwriting it.